src/test/kotlin/org/rust/cargo/commands/CargoTestCase.kt

```diff
@@ -0,0 +1,35 @@
+package org.rust.cargo.commands
+
+import com.intellij.openapi.projectRoots.SdkType
+import com.intellij.openapi.util.io.FileUtil
+import com.intellij.openapi.vfs.LocalFileSystem
+import com.intellij.testFramework.PlatformTestCase
+import org.rust.cargo.project.RustSdkType
+import org.assertj.core.api.Assertions.assertThat
+
+class CargoTestCase : PlatformTestCase() {
```

Comment:
It should be possible to use plain old `TestCase` here. I used `PlatformTestCase` because of `copyDirContentsTo` convenient method.